test/clusterTest.js

```diff
@@ -48,4 +48,16 @@ describe('Cluster', function() {
             });
         });
     }); // setTimeout
+
+    describe('getTimeout', function() {
+        it('should return the default timeout to 5\'000 ms', function(done) {
```

Comment:
Fix: should return the default timeout *to* 5\'000 ms.
Don't use escaped apostrophes as they obfuscate the code. Just write "5 seconds".